Plotly_test.ipynb

```diff
@@ -2361,7 +2361,7 @@
     }
```

Comment:
<p>Could we make this plot larger?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/test-v1/DeepReinforcementLearningNanodegree/pull/4/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>